src/hyperion/parameters/plan_specific/rotation_scan_internal_params.py

```diff
@@ -152,4 +152,4 @@ def get_data_shape(self) -> tuple[int, int, int]:
         size = (
             self.hyperion_params.detector_params.detector_size_constants.det_size_pixels
         )
-        return (self.experiment_params.get_num_images(), size.width, size.height)
+        return (self.experiment_params.get_num_images(), int(size.width), int(size.height))
```

Comment:
Should: Thanks for doing this fix but I actually recently fixed it in a nicer way in https://github.com/DiamondLightSource/dodal/pull/292 so I don't think we need the cast here and can rely on the typing system

Reply:
ok cool

Reply:
the NexusWriter expects a direct `tuple[int, int, int]` type, and `get_data_shape` function signature requires that as well. Which one should change?


![image](https://github.com/DiamondLightSource/hyperion/assets/56644812/fb52b185-cc81-4b92-95c0-7799ca286dd7)

Reply:
They are all correct, the change I linked fixed it such that `size.width` and `size.height` are now ints

Reply:
pulled in the change, resolved